surveys/promise-rejections/survey.md

```diff
@@ -0,0 +1,80 @@
+# Node.js Promise reject use case survey
+
+## Are you currently using Promises, async/await, a mix, or neither?
+
+Assume using as creating your own `new Promise`, `Promise.resolve`, and `async function`; or using libraries that produce Promises (that you use with `.then`/`.catch`, or with `.await`)
+
+(Multiple choice)
+
+  - [ ] Promises
+  - [ ] async/await
+  - [ ] Not writing Promise-based code or using Promise-based libraries
+
+## How are you handling rejections today?
+
+When consuming Promises, async functions or thenables, which of the options below do you use to handle rejections?
```

Comment:
Explain what a rejection is here? I'd assume a large number of people don't know what rejections actually are and assume they are just "async errors" that come from async functions.